.pre-commit-config.yaml

```diff
@@ -16,8 +16,24 @@ repos:
     hooks:
       - id: clang-format
         types_or: [c]
+  - repo: https://github.com/markdownlint/markdownlint.git
```

Comment:
I'd use https://github.com/DavidAnson/markdownlint-cli2/blob/main/.pre-commit-hooks.yaml
Mainly because it has support in a lot of IDEs + the config format is easier to use IMHO